releasing/plugin-commands-publishing/src/pack.ts

```diff
@@ -111,6 +113,8 @@ export async function handler (
       [path.join(dir, 'package.json')]: publishManifest as Record<string, unknown>,
     },
   })
+  // display order by name
+  globalInfo(`${chalk.blueBright('Tarball Contents')}\n${files.sort().join('\n')}`)
```

Comment:
Don’t send ANSI escape codes to the `@pnpm/logger`.

There are 2 possible ways to implement it:

* return the whole output from the function
* log the list of files as a debug message and render the output in `@pnpm/default-reporter`.

Reply:
done